app/assets/stylesheets/components/_buttons.scss

```diff
@@ -7,6 +7,7 @@
   background-color: black;
   border-color: black;
   box-shadow: none;
+  padding: $s35/2 4rem;
```

Comment:
there's a small risk that this could make some odd buttons elsewhere. `--primary` & `--cta` are mainly intended as color changes that could be applied to different size buttons. I'm not sure if we have any small `button--primary` but would be good to separate padding & anything with absolute sizing from the color styles.

Reply:
moving it into .button instead (to solve the "--primary & --cta are mainly intended as color" issue) and moving it out of this global context into the homepage specific .body--home selector in application.scss (to solve the "could make some odd buttons elsewhere issue")

i was assuming the wider buttons were desired in the design generally but this is definitely safer and can always be expanded later. thanks!